src/device/intrinsics/wmma.jl

```diff
@@ -37,6 +52,19 @@ const map_ptx_as_to_as_ty = Dict(
                                  "global" => AS.Global
                                 )
 
+ldst_ab_ops = ["m16n16k16", "m32n8k16", "m8n32k16"], ["a", "b"], ["f16", "u8", "s8"]
+ldst_cd_ops = ["m16n16k16", "m32n8k16", "m8n32k16"], ["c", "d"], ["f16", "f32", "s32"]
+ldst_int_ab_ops = ["m16n16k16"], ["a", "b"], ["u8", "s8"]
+ldst_int_cd_ops = ["m16n16k16"], ["c", "d"], ["s32"]
+ldst_subint_ab_ops = ["m8n8k32"], ["a", "b"], ["s4","u4"]
+ldst_bit_ab_ops = ["m8n8k128"], ["a", "b"], ["b1"]
+ldst_subint_cd_ops = ["m8n8k32", "m8n8k128"],  ["c", "d"], ["s32"]
```

Comment:
These three WMMA variants are not implemented here, right? I would just remove these lines to avoid confusion later on.